components/builder-api/src/metrics.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Centralized definition of all Builder API metrics that we
+//! wish to track.
+
+use std::borrow::Cow;
+use bldr_core::metrics;
+
+pub enum Counter {
+    GithubEvent
```

Comment:
I hate to be the pedant, but technically this is `GitHub`, and is spelled that way in many other places across our code.

Reply:
Consistency is good... I'll change it.

Thank you for your pedantry 😄